appyx-navigation/common/src/commonMain/kotlin/com/bumble/appyx/navigation/composable/AppyxComponent.kt

```diff
@@ -56,189 +19,27 @@ internal val defaultExtraTouch = 48.dp
 fun <InteractionTarget : Any, ModelState : Any> ParentNode<InteractionTarget>.AppyxComponent(
     appyxComponent: BaseAppyxComponent<InteractionTarget, ModelState>,
     modifier: Modifier = Modifier,
-    draggables: ImmutableList<Draggable> = immutableListOf(appyxComponent),
```

Comment:
I think we should remove it or find a better implementation. If we accidentally pass an empty list it will stop working